### PR TITLE
Incorporated Feedback from content team, polish, change button

### PR DESCRIPTION
**Pull request by ApheleiaS**
## Description
As a Contributor,
I need to have all the relevant information for Open Source Hack,
so that I can participate in the program.

### Content Team Feedback
Isabel's Feedback
Foong's Feedback
Sakshi's Feedback
Jhalak's Feedback

### Button UI Changes
Screen Shot 2020-08-14 at 8 04 29 AM

Button Changes Needed: 
- Rounded Button
- Caribbean Blue (#54BCEB) 
- OnHover: Navy (#005c88)
_Note: Be careful not to change the css that will alter the buttons in the carousel. You may need to add a new class. The carousel buttons should not be changed._

### Update [Required]
- [x] Incorporate any changes listed in the feedback provided from the [Content Team on Zulip](https://anitab-org.zulipchat.com/#narrow/stream/216324-outreach-research/topic/Task.20Force.20-.20Open.20Source.20HackDay/near/203543919).
- [x] Update the "Available Issues" button in the **Open Source Hack Issues** section.
- [x] Move the **Open Source Hack Issues** section to above the **FAQ** section and below the **Eligibility** section.

### CODE REVIEW

File: open-source-hack/index.html [hunk 1]
```diff
@@ -455,6 +502,18 @@ <h4 class="title">Frequently Asked Questions</h4>
...
 								</div><!-- card -->
+
+								 <div class="card">
+									<div class="card-header" id="headingEight">
+										<a aria-controls="collapseEight" aria-expanded="false" class="collapsed" data-target="#collapseEight" data-toggle="collapse" href="#">I have some more questions related to Open Source Hack/AnitaB.org?</a>
+									</div>
+									<div aria-labelledby="headingEight" class="collapse" data-parent="#accordionExample" id="collapseEight">
+                                        <div class="card-body">
+											<p class="text">Ask us <a href="">here!</a></p>
+										</div>
```
Comment (mayburgos) [inline open-source-hack/index.html:515]: For the link of "Ask us here." Use the questions stream in zulip: https://anitab-org.zulipchat.com/#narrow/stream/223070-questions

**Review by mayburgos — CHANGES_REQUESTED** [review]
I think we're just missing one link.

File: open-source-hack/index.html (hunk 1 above)
Comment (ApheleiaS) [reply open-source-hack/index.html:515]: Done! Also updated the text a little!

**Review by mayburgos — APPROVED** [review]
Looks good to me!

**Pull request merged**